tools/make/tests.toml

```diff
@@ -44,13 +44,15 @@ rm -f Cargo.lock
 
 pwd = pwd
 set_env CARGO_TARGET_DIR ${pwd}/target
+set_env CARGO "cargo --config=../patch.toml"
 
-exec --fail-on-error cargo --config patch.toml run -p demo_baked
+exec --fail-on-error cargo --config patch.toml run --release -p demo_baked
```

Comment:
If you just want to check that these build, don't use release mode. It'll unnecessarily slow down CI.

Reply:
I need `--release` in order for data slicing to work

Reply:
I need at least one of them to be `--release` so it makes sense for all of them to be `--release`. It is not slow to build these in `--release`. Datagen is slow in `--release` because it has a large amount of dependencies so LTO takes forever.

Reply:
`--release` doesn't have debug symbols, so if any of these fail they'll be annoying to debug.

Reply:
okay I made the `-local` task run in debug and `-cratesio` run in release.